setup.py

```diff
@@ -11,10 +12,20 @@
     'doc': ['sphinx', 'sphinx_rtd_theme'],
 }
 
-
 def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
+ext_modules = [
+    Extension(
+        "snap7.__dummy__",
```

Comment:
so this is still required to trigger the binary wheel creation? or is it for another reason?

Reply:
That's right, auditwheel cannot determine dependencies without this. https://github.com/pypa/auditwheel#limitations